tutorecommerce/templates/ecommerce/build/ecommerce/Dockerfile

```
@@ -79,6 +80,9 @@ RUN cd /openedx/requirements/ \
 {% for extra_requirement in ECOMMERCE_EXTRA_PIP_REQUIREMENTS %}RUN --mount=type=cache,target=/openedx/.cache/pip,sharing=shared,uid=${APP_USER_ID} pip install '{{ extra_requirement }}'
 {% endfor %}
 
+RUN atlas pull --repository="{{ ATLAS_REPOSITORY }}" --branch="{{ ATLAS_REVISION }}" {{ ATLAS_OPTIONS }} translations/ecommerce/ecommerce/conf/locale:ecommerce/conf/locale
```

@shadinaif There's a very new addition to atlas:

 - https://github.com/openedx/openedx-atlas/pull/49

Would you like to implement it in this pull request or create a new one after this one is merged?

```suggestion
RUN atlas pull --repository="{{ ATLAS_REPOSITORY }}" --revision="{{ ATLAS_REVISION }}" {{ ATLAS_OPTIONS }} translations/ecommerce/ecommerce/conf/locale:ecommerce/conf/locale
```

cc: @Faraz32123 @brian-smith-tcril

---

@shadinaif could you please accept that suggestion?

---

Meanwhile, I've upgraded the ecommerce PR:

 - https://github.com/openedx/ecommerce/pull/4108

We need to merge that one first.

---

updated and rebased, waiting for [4108](https://github.com/openedx/ecommerce/pull/4108) to be merged